contracts/NodeDelegator.sol

```diff
@@ -338,6 +395,168 @@ contract NodeDelegator is INodeDelegator, LRTConfigRoleChecker, PausableUpgradea
         ISSVNetwork(SSV_NETWORK_ADDRESS).registerValidator(publicKey, operatorIds, sharesData, amount, cluster);
     }
 
+    /// @notice Requests a withdrawal of liquid staking tokens (LST) from EigenLayer's underlying strategy.
+    /// Is only callable by the `LRTDepositPool` contract.
+    /// @param strategyAddress the address of the EigenLayer LST strategy to withdraw from
+    /// @param strategyShares the amount of EigenLayer strategy shares to redeem
+    /// @param staker the address of the staker requesting the withdrawal.
+    function requestWithdrawal(
+        address strategyAddress,
+        uint256 strategyShares,
+        address staker
+    )
+        external
+        onlyDepositPool
+        whenNotPaused
+    {
+        // request the withdrawal of the LST asset from EigenLayer
+        bytes32 withdrawalRoot = _requestWithdrawal(strategyAddress, strategyShares);
+
+        // store a mapping of the returned withdrawalRoot to the staker withdrawing
+        withdrawalRequests[withdrawalRoot] = staker;
```

Comment:
Let's check that this is empty first. I know it's theoretically a hash and impossible to duplicate, but it just feels right to ensure we aren't taking over someone else's withdraw request

Reply:
I'll added a duplicate request withdrawal check